homeassistant/components/wirelesstag/__init__.py

```diff
@@ -56,15 +58,33 @@
 )
 
 
-class WirelessTagPlatform:
+class WirelessTagPlatform(HomeAssistantView):
     """Principal object to manage all registered in HA tags."""
 
+    requires_auth = False
+    url = "/api/wirelesstag/{event}"
+    name = "api:wirelesstag"
+
     def __init__(self, hass, api):
         """Designated initializer for wirelesstags platform."""
         self.hass = hass
         self.api = api
         self.tags = {}
-        self._local_base_url = None
+
+    async def post(self, request, event):
+        """Accept the POST."""
+        data = None
+        try:
+            data = await request.json()
+        except ValueError as vex:
+            _LOGGER.error("Invalid JSON: %s", str(vex))
```

Comment:
You should return if it's invalid. Things are automatically converted to strings:

```suggestion
            _LOGGER.error("Invalid JSON for %s: %s", event, vex)
            return
```